sklearn/naive_bayes.py

```diff
@@ -951,3 +964,182 @@ def _joint_log_likelihood(self, X):
         jll += self.class_log_prior_ + neg_prob.sum(axis=1)
 
         return jll
+
+
+class CategoricalNB(BaseDiscreteNB):
+    """
+    Naive Bayes classifier for categorical features
+
+    The categorical Naive Bayes classifier is suitable for classification with
+    discrete features, that are categorically distributed. Each feature is sampled
+    by its own categorical distribution.
```

Comment:
categorical distribution being a smoothed multinomial?

Reply:
First of all, thank you very much for your detailed review. I really appreciate it!

I changed the description to [here](https://github.com/scikit-learn/scikit-learn/pull/12569/files#diff-dff46bc51c084c731bcbe4cb9b6aa243R996). Does this solve your remark?